projects/com.oracle.truffle.llvm.test/src/com/oracle/truffle/llvm/test/TestGCCSuite.java

```diff
@@ -91,4 +92,8 @@ public void test() {
         }
     }
 
+    private static int truncate(int retValue) {
+        return retValue & (UNSIGNED_BYTE_MAX_VALUE - 1);
```

Comment:
nitpicking: actually, the unsigned byte max value is 0xff == 255, so I'd change the constant to that.